tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -297,6 +297,21 @@ def testee(a: cases.IJKField, b: int32) -> cases.IJKField:
     cases.verify(cartesian_case, testee, a, b, out=out, ref=ref)
 
 
+@pytest.mark.uses_tuple_args
+def test_double_use_scalar(cartesian_case):
+    # TODO(tehrengruber): This should be a regression test on ITIR level, but tracing doesn't
+    #  work for this case.
```

Comment:
see my comment in slack and clarify here if needed: "tracing doesn't work" means it would inline?

Reply:
wouldn't it work if you put the calculation of `tmp` in a separate `fundef`?